x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -38,84 +67,198 @@ export const SetupPage: React.FunctionComponent<{
     }
   };
 
-  const content =
-    missingRequirements.includes('tls_required') ||
-    missingRequirements.includes('api_keys') ||
-    missingRequirements.includes('encrypted_saved_object_encryption_key_required') ? (
-      <>
-        <EuiSpacer size="m" />
-        <EuiIcon type="lock" color="subdued" size="xl" />
-        <EuiSpacer size="m" />
-        <EuiTitle size="l">
-          <h2>
+  if (
+    !missingRequirements.includes('tls_required') &&
+    !missingRequirements.includes('api_keys') &&
+    !missingRequirements.includes('encrypted_saved_object_encryption_key_required')
+  ) {
+    return (
+      <WithoutHeaderLayout>
+        <EuiPageBody restrictWidth={548}>
+          <EuiPageContent
+            verticalPosition="center"
+            horizontalPosition="center"
+            className="eui-textCenter"
+            paddingSize="l"
+          >
+            <EuiSpacer size="m" />
+            <EuiIcon type="lock" color="subdued" size="xl" />
+            <EuiSpacer size="m" />
+            <EuiTitle size="l">
+              <h2>
+                <FormattedMessage
+                  id="xpack.ingestManager.setupPage.enableTitle"
+                  defaultMessage="Enable Fleet"
+                />
+              </h2>
+            </EuiTitle>
+            <EuiSpacer size="xl" />
+            <EuiText color="subdued">
+              <FormattedMessage
+                id="xpack.ingestManager.setupPage.enableText"
+                defaultMessage="In order to use Fleet, you must create an Elastic user. This user can create API keys
+        and write to logs-* and metrics-*."
+              />
+            </EuiText>
+            <EuiSpacer size="l" />
+            <EuiForm>
+              <form onSubmit={onSubmit}>
+                <EuiButton fill isLoading={isFormLoading} type="submit">
```

Comment:
can be combined to be an `onClick` on the EuiButton instead of adding form element